### DISCUSSION TITLE
Syntax for multiple source files

### DISCUSSION THREAD

**Discussion by mbarritt** [coddingtonbear/obsidian-itinerary#15] [Q&A] [answered]
The readme says that multiple source files work, but I cannot hit upon the correct syntax for listing more than one. Can you give an example of the correct syntax in the "source:..." line for referencing more than one file as the source?

Thanks

**Comment by coddingtonbear** [comment, marked as answer]
Like most obsidian plugins using the source block, the data there is YAML.  YAML allows a variety of methods of specifying an array, but what I'd recommend you do is something like:

```
source:
- "/path/to/first/file"
- "/path/to/second/file"
```

**Comment by mbarritt** [comment]
That works. Thanks.

Following on from this; is there a way to make it a dynamic array pulling in files that match contain a certain tag? (Allowing me to have content for the itinerary spread ad hoc through a wide variety of notes all marked with a particular tag.) 

I've tried to do this by creating a note that pulls in itinerary code blocks wherever they are (marking these code blocks with a particular block id) and then referencing this compiled doc as a source. So far no luck. 

Any ideas? Thanks again.

**Reply by coddingtonbear** [reply]
I'm afraid nothing like that exists at the moment, but if you were thinking of contributing such a feature, I'd be glad to talk over what that might look like.

**Comment by jane-t** [comment]
Sorry to trouble you, but I am new to Obsidian and trying to include my trip pages in to the source page.

I have tried a few different options in source and have not been able to get the path correct.

The page path in my "Obsidian" vault looks like this

d:\Obsidian\Holidays\Argentina and Antarctica\11-26 Getting There.md

I have tried
```
itinerary
initialDate: 2022-11-26
source:
  - "/Holidays/Argentina and Antarctica/11-26 Getting There"
```
but no events appear in the calendar.

If I put the calendar on the getting there page it correctly shows the events, please could you point me in the correct direction,  thank you.
